Package.swift

```diff
@@ -55,3 +55,8 @@ let package = Package(
         .target(name: "CodableSample", dependencies: ["AWSLambdaRuntime"]),
     ]
 )
+
+#if compiler(>=5.5) && $AsyncAwait
+package.targets.first(where: { $0.name == "AWSLambdaRuntimeCore" })?
+    .dependencies.append(.product(name: "_NIOConcurrency", package: "swift-nio"))
```

Comment:
you can always depend on it, `_NIOConcurrency` just doesn't export anything if A/A isn't on. But up to you